src/components/Image/Image.tsx

```diff
@@ -18,16 +20,27 @@ const Image = ({ src, thumbnail, alt }: ImagePropsType) => {
         src={thumbnail}
       />
       <img
-        onLoad={() => {
+        onLoad={(e) => {
           setIsLoaded(true);
         }}
+        onError={(e) => {
+          const element = e.currentTarget as HTMLImageElement;
+          element.src = fallback;
+        }}
         className={`image image--full ${
           isLoaded ? 'opacity-100' : 'opacity-0'
         }`}
         alt={alt ?? ''}
         src={src}
       />
-    </>
+      <img
+        className="invisible"
+        src={src}
+        onLoad={(e) => {
+          return setHeight(e.currentTarget.height);
```

Comment:
It's a weird hack to display images that height is not known yet (and I have to specify this because of absolute positioning). 
I have no idea how to do this differently without losing the effect we currently have (loading blurred thumbnail etc)

Reply:
Please add it ot our agenda

Reply:
added